tutorials/Kling_Gupta_efficiency.ipynb

```diff
@@ -0,0 +1,392 @@
+   "id": "5d527e93",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "obs = 10 * np.random.random((50, 50))  # Generate obs sample with mean of about 5.0\n",
+    "obs = xr.DataArray(\n",
+    "    data=obs, \n",
+    "    dims=[\"time\", \"x\"],\n",
+    "    coords={\"time\": pd.date_range(\"2023-01-01\", \"2023-02-19\"), \"x\": np.arange(0, 50)}\n",
+    ")\n",
+    "\n",
+    "# Create forecasts\n",
+    "fcst1 = obs - 4      # Generate Forecast system 1 with with mean of about 1.0\n",
+    "\n",
+    "fcst2 = 4.5 + np.random.random((50, 50))  # Generate Forecast system 2 with with mean of about 5.0\n",
+    "fcst2 = xr.DataArray(\n",
+    "    data=fcst2, \n",
+    "    dims=[\"time\", \"x\"],\n",
+    "    coords={\"time\": pd.date_range(\"2023-01-01\", \"2023-02-19\"), \"x\": np.arange(0, 50)}\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "51fb2970",
+   "metadata": {},
+   "source": [
+    "Let us the check mean and standard devaition of the generated observations and forecast samples."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "185ac7e7-2456-45e3-a7a8-6b900470a892",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(<xarray.DataArray ()> Size: 8B\n",
+       " array(4.98878265),\n",
+       " <xarray.DataArray ()> Size: 8B\n",
+       " array(2.90827267))"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "obs.mean(), obs.std()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "34462f86-062a-469b-801f-247c4bb00692",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(<xarray.DataArray ()> Size: 8B\n",
+       " array(0.98878265),\n",
+       " <xarray.DataArray ()> Size: 8B\n",
+       " array(2.90827267))"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "fcst1.mean(),fcst1.std()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "7ad5b22f-c5b1-4ede-9a91-5cd2f43038b2",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(<xarray.DataArray ()> Size: 8B\n",
+       " array(4.99478572),\n",
+       " <xarray.DataArray ()> Size: 8B\n",
+       " array(0.2883624))"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "fcst2.mean(),fcst2.std()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6a2c850b-6082-4f65-a892-9dbafdbf73c8",
+   "metadata": {},
+   "source": [
+    "The mean of obs and fcst2 are similiar, but theirs standard deviation are quite different. On the other hand, the standard deviation of obs and fcst1 are similiar, but their means are quite different. This can be seen in the following figure."
```

Comment:
```suggestion
    "The mean of `obs` and `fcst2` are similiar, but their standard deviations are quite different. On the other hand, the standard deviation of `obs` and `fcst1` are similiar, but their means are quite different. This can be seen in the following figure."
```
1. Should backticks be added as above?
2. Minor grammar issue in: "but theirs standard deviation are quite different" (if my suggestion of "but their standard deviations are quite different" isn't right, please modify correctly).